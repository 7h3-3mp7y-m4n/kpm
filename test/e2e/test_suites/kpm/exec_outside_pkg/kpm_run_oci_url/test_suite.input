kpm run oci://localhost:5001/test/kcl1