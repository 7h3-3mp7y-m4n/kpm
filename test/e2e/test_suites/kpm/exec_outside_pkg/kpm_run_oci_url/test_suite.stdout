kpm: the lastest version '0.0.1' will be pulled.
kpm: pulling '/test/kcl1:0.0.1' from 'localhost:5001/test/kcl1'.
kpm: downloading 'test/k8s:1.27' from 'localhost:5001/test/k8s:1.27'.
The_first_kcl_program: Hello World!